### PR TITLE
Twilio Video SDK Podspec

### PR DESCRIPTION
**Pull request by ptankTwilio**

### CODE REVIEW

**Comment by ceaglest** [comment]
This PR looks good to me, but the Podfile in the sample app needs to be updated to look like:

```
source 'https://github.com/twilio/cocoapod-specs'

target 'VideoQuickStart' do
    pod 'TwilioVideo', '1.0-preview1'
end
```

File: TwilioVideo/1.0.0-preview1/TwilioVideo.podspec [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  s.description  = "The Twilio Video SDK lets you embed real-time voice and video into your iOS apps."
+  s.homepage     = "http://www.twilio.com/video"
+  s.platform     = :ios, "8.1"
+  s.license      = { 
+    :type => "Commercial", 
+    :text => "Copyright 2011-2016 Twilio. All rights reserved. Use of this software is subject to the terms and conditions of the Twilio Terms of Service located at http://www.twilio.com/legal/tos"
+  }
+  s.author       = { "Twilio" => "help@twilio.com" }
+  s.source       = { :http    => "https://media.twiliocdn.com/sdk/ios/video/releases/1.0.0-preview1/twilio-video-ios-1.0.0-preview1.tar.bz2" }
+  s.frameworks            = "AudioToolbox", "VideoToolbox", "AVFoundation", "CoreTelephony", "GLKit", "CoreMedia", "SystemConfiguration"
```
Comment (paynerc) [inline TwilioVideo/1.0.0-preview1/TwilioVideo.podspec]: I am not sure that we need to specify all of this anymore.

Comment (ceaglest) [inline TwilioVideo/1.0.0-preview1/TwilioVideo.podspec]: The beauties of a dynamic framework. @ptankTwilio Can you update the PR to remove this line?

**Comment by ceaglest** [comment]
👍 merge once the url is live.

**Pull request merged**